default.nix

```diff
@@ -1,5 +1,30 @@
-# This default.nix is designed to be passed directly nixpkgs with something like:
-#   import <nixpkgs> (import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz))
-{ config   = import ./config.nix;
-  overlays = import ./overlays;
-}
\ No newline at end of file
+let
+  # Names nixpkgs pins (corrisponds to names of the nixpkgs/*.json files)
+  nixpkgs-names = ["release-18.08" "release-19.03" "release-19.09"];
+
+  # Name of the default nixpkgs-pin
+  default-nixpkgs-name = "release-19.09";
+
+  # These are from lib.attrset but we don't want to have to import a nixpkgs just to get them
+  nameValuePair = name: value: { inherit name value; };
+  genAttrs = names: f: builtins.listToAttrs (builtins.map (n: nameValuePair n (f n)) names);
+
+in {
+  # nixpkgsArgs is designed to be passed directly nixpkgs with something like:
+  #   import <nixpkgs> (import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz)).nixpkgsArgs
+  nixpkgsArgs = {
```

Comment:
This is a breaking change for users, right? Trivial fix, so we can probably just eat it, but worth noting.